src/node/signer.ts

```diff
@@ -15,32 +15,24 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import { serializeTags, streamSigner } from 'arbundles';
-import { randomBytes } from 'node:crypto';
 import { Readable } from 'node:stream';
 
-import { JWKInterface } from '../common/jwk.js';
+import { TurboDataItemAbstractSigner } from '../common/signer.js';
 import {
   DataItemOptions,
   StreamSizeFactory,
-  TurboDataItemSigner,
+  TurboDataItemSignerParams,
   TurboLogger,
   TurboSigner,
 } from '../types.js';
-import { fromB64Url, toB64Url } from '../utils/base64.js';
+import { fromB64Url } from '../utils/base64.js';
 
-export class TurboNodeArweaveSigner implements TurboDataItemSigner {
-  protected privateKey: JWKInterface;
+export class TurboNodeArweaveSigner extends TurboDataItemAbstractSigner {
   protected signer: TurboSigner;
   protected logger: TurboLogger;
```

Comment:
these are no longer needed if we are extending the parent class

Reply:
handled in [466a705](https://github.com/ardriveapp/turbo-sdk/pull/90/commits/466a7055deaa674f6352793ba3caa9da1f511cde)